app/services/event.service.js

```diff
@@ -301,7 +422,7 @@
 
       //Default params
       var defaultParams = {
-        //fields: 'id,label,date.start_month,date.start_day,date.start_time,date.end_time,image.image_styles.large,image.alt,uri,body_trimmed,summary,clndrDate,timezone,venue_name',
```

Comment:
Are we 100% sure your edits to this return the fields we need? I recall that I had to comment out fields, to ensure we were getting all we needed.